utils/ratelimits.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, name='global', expire_time=timedelta(0, 1, 0)):
     def __getitem__(self, item):
         db = get_redis()
         c = db.hgetall(f'ratelimit-cache:{self.id}:{item}')
-        now = datetime.now()
+        now = datetime.utcnow()
         previous = datetime.strptime(c['timestamp'], '%Y-%m-%d %H:%M:%S.%f')
         expiry = datetime.strptime(c['expire_time'], '%H:%M:%S')
         expiry = expiry - datetime(1900, 1, 1)
```

Comment:
Epoch time starts from 01 Jan 1970, btw. Not sure if this is used anywhere else but if it's a factor in rate-limiting then it could look weird in other languages.